src/main/java/net/minestom/server/MinecraftServer.java

```diff
@@ -804,4 +832,8 @@ private static void checkInitStatus(@Nullable Object object) {
                         "if you are developing an extension be sure to retrieve them at least after Extension#preInitialize");*/
     }
 
+    @Override
+    public @NotNull Audience audience() {
+        return getConnectionManager();
+    }
```

Comment:
Should be implementing `Iterable` variant (with above interface change), and return an `Iterable` consisting of all online players and the server console.